extensions/amp-a4a/0.1/a4a-variable-source.js

```diff
@@ -27,6 +27,7 @@ import {user} from '../../../src/log';
 const WHITELISTED_VARIABLES = [
   'RANDOM',
   'COUNTER',
+  'SUM',
```

Comment:
can you sort your new function in alphabetic order?
same below

Reply:
Done. I wasn't sure how to sort one item in an unsorted list, so I just sorted everything.